### ISSUE TITLE
Add ToggleSwitch Focus keyboard action

### ISSUE THREAD

**Issue by lmseddik** [MahApps/MahApps.Metro#368]
The toggle switch control should have the keyboard focus (istabstop=true) and changes his state `IsChecked = ! IsChecked` when the user presses the spacebar like (the checkbox or the radiobutton)
